roles/container-engine/containerd/tasks/main.yml

```
@@ -0,0 +1,43 @@
+---
+- name: containerd | Install required packages
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - libseccomp2
+    - ethtool
+
```

To expand this to cover some non-debian distributions, consider:
```suggestion
  when: ansible_distribution in ["Debian","Ubuntu"]

- name: containerd | Install required packages (RPM)
  yum:
    name: "{{ item }}"
    state: present
  with_items:
    - libseccomp
    - ethtool
  when: ansible_distribution in ["CentOS","RedHat"]
```

If the host is RHEL/CentOS Atomic I guess direct containerd usage is not an option and this conflict should be caught somewhere?

I did not find anything else Debian-specific in the PR.

---

Thanks for the suggestions, I included those. Don't have anything RHEL-ish to test on right now so can't really improve on that.